pkg/storaged/lvm2/volume-group.jsx

```diff
@@ -67,6 +68,7 @@ function vgroup_rename(client, vgroup, card) {
         ],
         Action: {
             Title: _("Rename"),
+            Danger: contains_rootfs(client, vgroup.path) ? _("This volume group contains the root filesystem. Renaming it might require further changes to the bootloader configuration or kernel command line.") : null,
```

Comment:
This added line is not executed by any test.

Reply:
I don't understand this. The tests cover both scenarios, renaming the rootfs, and renaming something else.